config/model_configs/slabplanet_eisenman.yml

```diff
@@ -0,0 +1,17 @@
+run_name: "slabplanet_eisenman"
+surface_setup: "PrescribedSurface"
+moist: "equil"
+vert_diff: "true"
+rad: "gray"
+energy_check: true
+mode_name: "slabplanet_eisenman"
```

Comment:
should this be "model_name"?

Reply:
We call the `slabplanet`, `AMIP` and now `slabplanet_eisenman` setups modes (it was unique enough from the other tags, but we can change this in the future if needs be :) )